src/rt/sched/cown.h

```diff
@@ -359,12 +370,18 @@ namespace verona::rt
       yield();
     }
 
+    void would_block_in_io()
```

Comment:
nit: name doesn't imply it's setting things. Perhaps something like `set_block_io` or something?